modules/auth0-auth-google/main.tf

```diff
@@ -2,7 +2,7 @@ resource "auth0_connection" "google_oauth2" {
   name                 = "google-oauth2"
   strategy             = "google-oauth2"
   is_domain_connection = var.is_domain_connection
-  enabled_clients      = var.enabled_clients
+  # enabled_clients      = var.enabled_clients
```

Comment:
is it ok to have this commented, if no need just remove with corresponding variable declaration, if you will use this in future have some comment and also comment out variable declaration